### PR TITLE
Add package source type for installation of bridges

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch adds a provision to specify the type of
package source to be considered for installation.

Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @brainfunked @nthomas-redhat @shtripat Please review

File: tendrl/node_agent/flows/import_cluster/ceph_help.py [hunk 1]
```diff
@@ -5,10 +5,18 @@
 
 
 def import_ceph(integration_id):
-    name = "git+https://github.com/Tendrl/ceph_integration.git@v1.2"
-    attributes = {"name": name}
-    attributes["editable"] = "false"
-    ansible_module_path = "core/packaging/language/pip.py"
+    attributes = {}
+    if tendrl_ns.config.data['package_source_type'] == 'pip':
+        name = "git+https://github.com/Tendrl/ceph_integration.git@v1.2"
```
Comment (r0h4n) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: the repo is called "ceph-integration"

File: tendrl/node_agent/flows/import_cluster/gluster_help.py [hunk 2]
```diff
@@ -5,10 +5,18 @@
 
 
 def import_gluster(integration_id):
-    name = "git+https://github.com/Tendrl/gluster_integration.git@v1.2"
-    attributes = {"name": name}
-    attributes["editable"] = "false"
-    ansible_module_path = "core/packaging/language/pip.py"
+    attributes = {}
+    if tendrl_ns.config.data['package_source_type'] == 'pip':
+        name = "git+https://github.com/Tendrl/gluster_integration.git@v1.2"
```
Comment (r0h4n) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: the repo is called "gluster-integration"

File: tendrl/node_agent/flows/import_cluster/ceph_help.py (hunk 1 above)
Comment (nnDarshan) [inline tendrl/node_agent/flows/import_cluster/ceph_help.py]: Done

File: tendrl/node_agent/flows/import_cluster/gluster_help.py (hunk 2 above)
Comment (nnDarshan) [inline tendrl/node_agent/flows/import_cluster/gluster_help.py]: Done

**Pull request merged**